blackbox/docs/config/cluster.rst

```diff
@@ -1098,3 +1098,37 @@ Metadata gateway
   because you only want the cluster state to be recovered once all nodes are
   started. However, the value must be bigger than the half of the expected
   number of nodes in the cluster.
+
+.. _s3-credentials:
```

Comment:
Is there a reason why it was brought back? Those settings cannot be via crate.yaml any longer.

Reply:
@kovrus I will have to look at this together with @mechanomi.